test/functional/rspec/spec/regression/kubernetes_context/kc_test_case_01_spec.rb

```diff
@@ -0,0 +1,46 @@
+# Test case 1: Install kubernetes module, stage and converge cluster
+
+require './lib/dtk_common'
+require './lib/assembly_and_service_operations_spec'
+require './lib/dtk_cli_spec'
+
+remote_module = 'kubernetes:kubernetes'
+remote_module_location = '~/dtk/modules/tests/kubernetes'
+version = 'master';
```

Comment:
It would be better if we test with specific version (we can omit version from the install command and it will install the latests available module). This way, we ensure that what is tagged as latest version really works (rather than testing on master which can be subject to frequent changes/breaks). `master` is living on the bleeding edge :)